Demo/Sources/Views/LongPressPlaybackView.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import PillarboxPlayer
+import SwiftUI
+
+#if os(iOS)
+private struct LongPressView<Content>: View where Content: View {
+    @GestureState private var isLongPressing = false
+    @State private var timer: Timer?
+
+    let minimumPressDuration: TimeInterval
+    @ViewBuilder let content: () -> Content
+    let action: (_ finished: Bool) -> Void
+
+    var body: some View {
+        content()
+            .simultaneousGesture(longPressGesture())
+            .onChange(of: isLongPressing) { isChanged in
+                timer?.invalidate()
+                if !isChanged {
+                    action(true)
+                } else {
+                    timer = Timer.scheduledTimer(withTimeInterval: minimumPressDuration, repeats: false) { _ in
+                        action(false)
+                    }
+                }
+            }
+    }
+
+    init(minimumPressDuration: TimeInterval = 1, content: @escaping () -> Content, action: @escaping (_ finished: Bool) -> Void) {
+        self.minimumPressDuration = minimumPressDuration
+        self.content = content
+        self.action = action
+    }
+
+    private func longPressGesture() -> some Gesture {
+        LongPressGesture(minimumDuration: .infinity, maximumDistance: .infinity)
+            .updating($isLongPressing) { value, state, transaction in
+                state = value
+                transaction.animation = .easeIn(duration: 2.0)
+            }
+    }
+}
+
+private struct LongPressPlaybackInfoView: View {
+    var body: some View {
+        HStack {
+            Text("x 2")
+                .font(.footnote)
+                .bold()
+            HStack(spacing: 3) {
+                ForwardArrowView()
+                ForwardArrowView()
```

Comment:
Likely easier to either:

- [Inline](https://www.hackingwithswift.com/quick-start/swiftui/how-to-insert-images-into-text) image in text directly.
- Use Unicode characters, for example ≫ (U+226B) and ≪(U+226A).

Reply:
I have chosen the inline approach.